test/WasmSpec/testsuite/memory.wast

```diff
@@ -64,7 +64,7 @@
 )
 
 (module (memory 0 0) (data (i32.const 0) ""))
-(module (memory 0 0) (data (i32.const 1) ""))
+(module (memory 1 1) (data (i32.const 0x10000) ""))
```

Comment:
This test was updated in the spec repo because it was actually invalid, where even empty data/element segment must have a valid offset as well.